sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -242,10 +244,10 @@ internal class BitmapWidgetRenderer(
         GLES20.GL_TEXTURE_WRAP_T,
         GLES20.GL_CLAMP_TO_EDGE.toFloat()
       )
-      GLUtils.texImage2D(GLES20.GL_TEXTURE_2D, 0, it, 0)
+      GLUtils.texImage2D(GLES20.GL_TEXTURE_2D, 0, bitmap, 0)
       GLES20.glBindTexture(GLES20.GL_TEXTURE_2D, 0)
 
-      bitmap = null
```

Comment:
Do we remove the bitmap reference any time with the current fix?

Reply:
No, the changes in this PR mean that the Bitmap is retained throughout the lifetime of the BitmapWidgetRenderer